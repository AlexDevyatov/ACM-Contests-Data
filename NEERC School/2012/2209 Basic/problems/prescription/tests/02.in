khoukse_k_1012_tenth_december_endoscopy
