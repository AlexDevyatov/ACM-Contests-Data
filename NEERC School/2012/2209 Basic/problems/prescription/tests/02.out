house__1012_tenth_december_endoscopy
